private/buf/bufformat/formatter_test.go

```diff
@@ -67,48 +62,45 @@ func testFormatProto3(t *testing.T) {
 }
 
 func testFormatNoDiff(t *testing.T, path string) {
-	ctx := context.Background()
-	runner := command.NewRunner()
-	moduleBucket, err := storageos.NewProvider().NewReadWriteBucket(path)
-	require.NoError(t, err)
-	module, err := bufmodule.NewModuleForBucket(ctx, moduleBucket)
-	require.NoError(t, err)
-	readBucket, err := Format(ctx, module)
-	require.NoError(t, err)
-	require.NoError(
-		t,
-		storage.WalkReadObjects(
-			ctx,
-			readBucket,
-			"",
-			func(formattedFile storage.ReadObject) error {
-				originalPath := formattedFile.Path()
-				if strings.HasPrefix(filepath.Base(originalPath), "option_name") {
-					// TODO: Temporarily skipping this test since it's
-					// due to a bug in protocompile.
-					//
-					// https://github.com/jhump/protocompile/pull/3
+	t.Run(path, func(t *testing.T) {
```

Comment:
Putting each file in its own test case provides much nicer output IMO. It also reduces the abruptness of a `require` assertion fail -- the test case that is immediately failed is just a single file instead of the entire function and all files.